### PR TITLE
Recipes for the v807 production

### PR DESCRIPTION
**Pull request by jhgoh**
Changes necessary for the v807 production

### CODE REVIEW

**Comment by jhgoh** [comment]
B tagging working points (unchanged, no effect on the production although it is written in the dataformat)
  * https://twiki.cern.ch/twiki/bin/view/CMS/BtagRecommendation80XReReco
Note: Tight WP of CvsB is even less than the loose WP, since this WP is more focused to remove light flavour jets

**Comment by jhgoh** [comment]
Electron regression & smearing (already implemented in the CATTools configuration)
  * (TBA) Deterministic smearing by ttH group https://gitlab.cern.ch/ttH/reference/blob/master/definitions/Moriond17.md
```

wget https://github.com/cms-egamma/RegressionDatabase/blob/master/SQLiteFiles/GED_80X_Winter2016/ged_regression_20170114.db?raw=true -O ged_regression_20170114.db
git clone -b Moriond17_gainSwitch_unc https://github.com/ECALELFS/ScalesSmearings.git
cd ../../..
```

**Comment by jhgoh** [comment]
brilcalc have been updated to 1.1.7 from 1.0.3. Change of normtag location had to be updated as well.
PR to be done in https://github.com/cms-kr/hep-tools

**Comment by jhgoh** [comment]
CSV weight had to be updated.
https://gitlab.cern.ch/ttH/reference/blob/master/definitions/Moriond17.md#63-csv-scale-factors
A separate PR is to be made.

File: CatProducer/python/patTools/egmVersionedID_cff.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ def enableElectronVID(process):
     electron_ids = [
         'cutBasedElectronHLTPreselecition_Summer16_V1_cff',
         'cutBasedElectronID_Summer16_80X_V1_cff',
-        'heepElectronID_HEEPV70_cff',
+        'heepElectronID_HEEPV60_cff',
```
Comment (jshlee) [inline CatProducer/python/patTools/egmVersionedID_cff.py]: is V60 correct?

Comment (jhgoh) [reply CatProducer/python/patTools/egmVersionedID_cff.py]: who is using HEEP and provide proper recipe?
Still the default CMSSW is pointing to V60, additional recipe is needed to enable V70.

Comment (jshlee) [reply CatProducer/python/patTools/egmVersionedID_cff.py]: i think @jalmond

Comment (jhgoh) [reply CatProducer/python/patTools/egmVersionedID_cff.py]: I'll roll back this change to use HEEP70

**Comment by jhgoh** [comment]
runtest OK

**Comment by jhgoh** [comment]
Merging this PR.

**Pull request merged**